lib/widgets/utils/data_display.dart

```diff
@@ -64,7 +64,7 @@ class DataDecorator extends StatelessWidget {
       child: Container(
         padding: const EdgeInsets.all(8.0),
         decoration: BoxDecoration(
-          border: Border.all(color: Colors.grey),
+          border: Border.all(color: Color(0xff5a78a0)),
```

Comment:
we shouldn't be using the colors this way. Any change would get very painful we should try to find a way to use the theme to accomplish that, or at worst make a set of color constant available thoughout the app to avoid using hex code each time